src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/auth_challenge_nonce.rs

```diff
@@ -0,0 +1,35 @@
+use crate::prelude::*;
+
+uniffi::custom_newtype!(
+    DappToWalletInteractionAuthChallengeNonce,
+    Exactly32Bytes
+);
+
+#[derive(
+    Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Ord, PartialOrd, Hash,
+)]
+pub struct DappToWalletInteractionAuthChallengeNonce(pub Exactly32Bytes);
```

Comment:
missing doc

Reply:
There doesn't seem to be any documentation for the wallet interaction models. Perhaps we can take some time later to document them all?